strategy/measuring-impact.md

```diff
@@ -0,0 +1,148 @@
+# Human Cell Atlas Data Coordination Platform - Our Strategy
+
+## Measuring our Impact
+
+This document provides background context for the overall aims and the specific aims, including:
+* ***value assumptions*** that define why we believe our aims are the right aims, and
+* ***indicators for success*** that define how we assess our progress against these aims
+
+### Aim 1
+
+**Create a data resource that maximizes the value and use of Human Cell Atlas data across the scientific community.**
+
+#### Value Assumptions
+
+Scientists expect the DCP to be a comprehensive resource, containing all data that has been generated in their area of research interest. If they aren’t confident that all the data they need is present, if metadata they need to interpret the data are missing, or if the data quality is too low, they will download HCA data from other resources.
+
+Scientists will use raw data from the DCP and “output data” from DCP processing pipelines as long as it is well described, and includes rich, detailed information about how the data was generated (including sample descriptions and protocol information).
+
+HCA metadata and data will generate significant scientific impact for a variety of researchers (namely the spectrum from computational biologists to wet-lab biologists) if represented in an accessible manner according to their technical needs and capabilities.
+
+Data generators will deposit data in the DCP because the DCP provides valuable incentives, such as accession numbers that can be cited in publications.
+
+#### Indicators for Success
+
+| Specific aim | Item | Indicator |
+| -----------: | ---- | :-------- |
+| 1 | a. | All data used in the Human Cell Atlas or tissue-specific cellular atlases (e.g. brain atlas) is present in the DCP |
+|   | b. | Data deposited match known volumes and assay types where these are available, for example when listed in project funding awards (e.g. seed network RFAs, MRC development cell atlas awards, Human Cell Atlas Horizon 2020 awards) |
+|   | c. | The impact of DCP data is demonstrated in portals that utilise it |
+|   | d. | The impact of DCP data is demonstrated in publications that cite it |
+| 2 | a. | A high proportion of protocols used in the DCP are documented in detail in protocols.io |
```

Comment:
I would add another line here "2c. A high proportion of analytical workflows used in the DCP are available on dockstore.org."

Really important our pipelines are seen and used by the community